test/registry/finalizeExit.js

```diff
@@ -0,0 +1,245 @@
+/* eslint-env mocha */
+/* global assert contract */
+const fs = require('fs');
+
+const config = JSON.parse(fs.readFileSync('./conf/config.json'));
+const paramConfig = config.paramDefaults;
+
+const utils = require('../utils.js');
+const BigNumber = require('bignumber.js');
+
+contract('Registry', (accounts) => {
+  describe('Function: finalizeExit', () => {
+    const [applicant, challenger] = accounts;
+
+    let token;
+    let registry;
+
+    before(async () => {
+      const { registryProxy, tokenInstance } = await utils.getProxies();
+      registry = registryProxy;
+      token = tokenInstance;
+
+      await utils.approveProxies(accounts, token, false, false, registry);
+    });
+
+    it('should allow a listing to exit when no challenge exists', async () => {
+      const listing = utils.getListingHash('google.com');
+
+      const initialApplicantTokenHoldings = await token.balanceOf.call(applicant);
+
+      await utils.addToWhitelist(listing, paramConfig.minDeposit, applicant, registry);
+
+      const isWhitelisted = await registry.isWhitelisted.call(listing);
```

Comment:
I don't think you need to test `utils.addToWhitelist` here. It's not exactly wrong to do so, but it adds noise to the test.